tests/utils/locators.ts

```diff
@@ -1,15 +1,5 @@
 import { Locator, Page } from '@playwright/test';
 
-export const viewportLocator = ({
-  page,
-  viewportId,
-}: {
-  page: Page;
-  viewportId: string;
-}): Locator => {
-  return page.locator(`css=div[data-viewportid="${viewportId}"]`);
-};
-
 export const viewportSVGLocator = ({
```

Comment:
These will go nicely into the `ViewportPageObject`. 😊 That said maybe even there is an SVGPageObject too!